lib/model/card/quest/hispaniola_1_rumours_of_an_island.dart

```diff
@@ -1,8 +1,11 @@
+import 'package:treasure_of_the_high_seas/model/card/action/replace_action.dart';
```

Comment:
Can we only use relative paths for imports from within treasure?

Reply:
I was thinking we do relative imports within the main sections (model, test, screens) and package import when importing from a different main section - but could always do relative imports

Reply:
Meh, I tend to never look at imports since the IDE takes care of them for us... is it really a big deal where they come from? I've never looked at / worried about them before on any other project. Would rather not have the overhead of having to go and "tidy up" imports that I've asked IntelliJ to bring in for me. 

The IDE even auto-hides them for you!

![image](https://user-images.githubusercontent.com/5732536/79065945-fc28b700-7cab-11ea-80b6-55fe909db726.png)